### ISSUE TITLE
Wrong security guard use/unuse sentences in HL1 Training.

### ISSUE THREAD

**Issue by malortie** [SamVanheer/halflife-unified-sdk#432]
In map `t0a0d` of current HL1 game release, the security guard uses sentences:

```
BA_HAZ_OK0 holo/tr_ba_use
BA_HAZ_WAIT0 holo/tr_ba_unuse
```

In the Unified SDK, the guard uses the post-disaster sentences.

**Steps to reproduce**

1. Open the console and type the following:

   ```
   map t0a0d
   ```

2. Use/Unuse security guard
3. Security guard should respond with post-disaster voice lines (incorrect)

**In-game version**

```
Build type: Release
Release type: Pre-Alpha
Client:
  Version 1.0.0
  Branch: master
  Tag: CI build
  Commit Hash: f47c2f6a405aa33b386b63188067e1853484687e
Server:
  Version 1.0.0
  Branch: master
  Tag: CI build
  Commit Hash: f47c2f6a405aa33b386b63188067e1853484687e

WIP build not suited for use (testing only)
```

**Comment by malortie** [comment]
This happens because `CBarney::KeyValue` calls `CBaseMonster::KeyValue` instead of `CTalkMonster::KeyValue`, which causes the code to skip the assignment of Use/Unuse sentences.

https://github.com/SamVanheer/halflife-unified-sdk/blob/f47c2f6a405aa33b386b63188067e1853484687e/src/game/server/entities/NPCs/talkmonster.cpp#L1344-L1355

**Comment by SamVanheer** [comment]
Fixed by #442.
